mavlink/modules/flight_input_device.py

```diff
@@ -0,0 +1,34 @@
+from dronekit import connect 
+
+class FlightInputDevice: 
+
+    def __init__(self, address: "str"):
+        """
+        address: tcp address (e.g. "tcp:127.0.0.1:14550")
+        """
+        self.drone = connect(address, wait_ready = True)
+        assert self.drone is not None
+
+    def get_data(self) -> "dict": 
+        """
+        For now since the only output is to odometry worker,
```

Comment:
Do you think you can edit the docstring here so that it just explains what the function does. Something like `Returns odometry data from the drone`